code/modules/mod/mod_control.dm

```diff
@@ -399,10 +399,17 @@
 		core.on_attackby(attacking_item, user, params)
 	else if(istype(attacking_item, /obj/item/mod/skin_applier))
 		return ..()
+	else if(istype(attacking_item, /obj/item/holder/pai))
+		var/created_drone = FALSE
```

Comment:
You should make this a common proc on `/obj/item/holder/pai` or `/obj/item/mod/module/drone` to remove this duplicated code

Reply:
As discussed on discord, the pAI code is obsolete and should've been removed by me earlier, apologies